packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -400,22 +579,53 @@ export class LitDevTutorial extends LitElement {
       return;
     }
 
-    this._solved = true;
+    this._checkIndeterminate = false;
+    this._checkPassed = true;
+    this._isChecking = false;
+    this._clearCheckingTimeout();
+    this._requestSolvedCode = true;
     this._setProjectSrc(this._info.projectSrcAfter, true);
   }
 
+  private _clearCheckingTimeout() {
+    if (this._currentCheckTimeout) {
```

Comment:
If you want, can remove the check. Passing null [is silently handled by clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout#notes).